packages/pasteup/metrics.ts

```diff
@@ -0,0 +1,19 @@
+export type Metric = number;
+
+export interface LayoutMetrics {
+    gutter: Metric;
+    baseline: Metric;
+    column: Metric;
+}
+
+export interface Metrics {
+    layout: LayoutMetrics;
+}
+
+export const metrics: Metrics = {
+    layout: {
+        gutter: 20,
+        baseline: 12,
+        column: 60,
```

Comment:
According to [design.theguardian.com](https://design.theguardian.com/) columns are `60px` at desktop breakpoints (`wide`, `leftCol` and `desktop`), but `40px` at `tablet`. Should we account for this somehow?